demo/scripts/column-chart.js

```diff
@@ -138,22 +150,25 @@ function getChartData(timeFrame){
         subtitle: 'Drawn for you',
         numberAxisOrient: 'left', //todo: refactor onto y object
         hideSource: false,
-        x:{
-          series: {key:'date', label:'yearly'}
-        },
+        x:{series: {key:'date', label:'yearly'}},
         tickSize: 5,
         y: { series: ySeries} ,
         seriesLength: ySeries.length,
         groupDates: units[timeFrame] || ['quarterly', 'yearly'],
-        data: fixtures[timeFrame]
+        data: fixtures[timeFrame],
+        chartSubtype: timeFrame,
     };
 }
 
 module.exports = {
     getChartData: getChartData,
     init: function(){
-        var demos = ['year','yearWithNegative','years','yearsWithNegative','decade', 'month', 'multiple'];
-        //var demos = ['multiple'];
+        //var demos = ['year','yearWithNegative','years','yearsWithNegative','decade', 'month', 'multiple', 'stacked'];
+        var demos = [];
+        for(var k in fixtures){
+            demos.push(k);
+        }
+
```

Comment:
if refactoring better removing the intermediary stack data structure "demos" so written as e.g.? 

for (var demo in fixtures){
     ..........
     either initialisation code inline here or extracted to a method e.g. renderDemoChart(demo);... 
}